README.md

```diff
@@ -132,4 +134,12 @@ If you use [`@antfu/ni`](https://github.com/antfu/ni), you can also use the foll
 nr docs:build
 ```
 
+## Star History
+
+[![Star History Chart](https://api.star-history.com/svg?repos=nolebase/integrations&type=Date)](https://star-history.com/#nolebase/integrations&Date)
+
+## Project Status
+
+🚧 The project owner should go to [Repobeats](https://repobeats.axiom.co/) website and generate a URL.
```

Comment:
```suggestion
![Alt](https://repobeats.axiom.co/api/embed/422f7e142085eeb0c7ef854046101bf90927de91.svg "Repobeats analytics image")
```

Thanks, generated.

Reply:
and we can show commit heatmap, but I just search but get nothing appropriate (all we do is just replace key info related to the repo), maybe you can achieve this.

Reply:
```suggestion
![Repobeats analytics image](https://repobeats.axiom.co/api/embed/422f7e142085eeb0c7ef854046101bf90927de91.svg "Repobeats analytics image")
```

The `Alt` should be filled in to improve accessibility. 

Also, do we need to add these two state diagrams to `README.zh-CN.md` as well?

Reply:
Yes, you can add them to `README.zh-CN.md` as well.